build.gradle

```diff
@@ -336,6 +336,9 @@ startScripts {
   doLast {
     unixScript.text = unixScript.text.replace('PANTHEON_HOME', '\$APP_HOME')
     windowsScript.text = windowsScript.text.replace('PANTHEON_HOME', '%~dp0..')
```

Comment:
I believe this will still fail trying to generate the long classpath (which is then ignored) because the `set CLASSPATH=%APP_HOME%\lib\pantheon-1.0.0-SNAPSHOT.jar;...` is still too long. 
If I remove that `set CLASSPATH` line ( REM is not enough ), then things start to work.

Reply:
I am with you on that. 

A solution could be to ultimately replace `set CLASSPATH=%APP_HOME%\lib\pantheon-1.0.0-SNAPSHOT.jar;...` with `set CLASSPATH=%APP_HOME%\lib\*`. What do you think?